internal/integration/README.md

```
@@ -0,0 +1,193 @@
+    }
+  },
+  "expectedResponse": {
+    "code": 200,
+    "body": "{\"jsonrpc\": \"2.0\", \"id\": 1, \"result\": {...}}"
+  }
+}
+```
+
+## Running Tests
+
+### Run all integration tests:
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)